### ISSUE TITLE
Night Mode

### ISSUE THREAD

**Issue by DrkMako** [Nexus-Mods/Nexus-Mod-Manager#787]
EDIT (for clarity) 
this is a feature request
I've used NMM for years, and I always have issues with Vortex in one form or another, but i likes its color layout
otherwise the community edition has been a great thing (I appreciate that it is still receiving updates /ect)  and I've always recommended it over vortex for function
My only Issue I have had is the White on white styled menu is rather painful to my eyes, this basically causes severe eye strain and headaches if Im using it and messing around in it for to long.

so if possible, a future update with a night mode or dark themed mode would be highly appreciated, even if its an additional download or something.

thank you for your time and efforts

**Comment by djt454** [comment]
Agreed. I have a 4k monitor and the white is so bright I have to turn down screen brightness at the expense of other apps that have a dark mode which is much easier on the eyes.

**Comment by k4tonic** [comment]
Sorry for the thread revival but I would also love to see this implemented.
